### ISSUE TITLE
Improve repo structure

### ISSUE THREAD

**Issue by guerrero** [nathos/sass-textmate-bundle#82]
IMHO it would be nice to improve the structure of this repo or split it into two differents, one for Textmate and another for Sublime. The problem here is that `master` branch is  around 50 commits ahead of `sublime` and due to this fact syntax and completion files are totally different between them.

Another problems to solve:
- Reflect in `README.md` that there's a specific branch for Sublime 
- Use git tags for Package Control versions
- Add tests
- Add messages for Package Control

I'm ready to collaborate as a mantainer or contributor to help to implement this features.

**Comment by guerrero** [comment]
Ping @nathos

**Comment by nathos** [comment]
@guerrero: Agreed on all counts. A long-running `sublime` branch was a bad idea, and as they diverged it became increasingly difficult to accept pull requests. `cherry-pick`ing commits was a nightmare.

Given the install counts, I think it likely makes sense for the Sublime package to be its own repo at this point.

I'd also like to message existing users (via an update) to tell them to install the new package. I brought some of this up a long while back in #58. According to wbond, it *might* be possible to make the transition rather seamless.

The package has been in a rather tenuous state for quite some time, and I've been hesitant to make large changes to it for fear of disrupting the work of many people. I think it'd be great if you would like to spearhead this effort. Whether that means creating a new project, adding you as a collaborator, or even transferring this repo to you, I'm open to all options.

**Comment by guerrero** [comment]
Nice, I want to do some research to find the better solution for the sublime branch problem before starting to contribute to this repo if it's ok with you. Thanks!
